src/Simplify.cpp

```diff
@@ -525,7 +538,11 @@ class Simplify : public IRMutator {
         const Select *select_a = a.as<Select>();
         const Select *select_b = b.as<Select>();
 
-        if (const_int(a, &ia) &&
+        if (is_error_call(call_a)) {
```

Comment:
Though, it looks like this was the old behavior too. I'm surprised this worked at all... signed_integer_overflow was a fix for a bug I was having that surely would break the code?